<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandDialog</class>
 <widget class="QDialog" name="CommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CANBUS Command</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="passwordPage">
      <property name="title" stdset="0">
       <string>Password</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelPassword">
         <property name="text">
          <string>Enter Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submitButton">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="commandPage">
      <property name="title" stdset="0">
       <string>Send CANBUS Command</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="labelCommand">
         <property name="text">
          <string>Enter CANBUS Command (5 Bytes)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="commandLayout">
         <item>
          <widget class="QLineEdit" name="byte1LineEdit">
           <property name="inputMask">
            <string>HH</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="byte2LineEdit">
           <property name="inputMask">
            <string>HH</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="byte3LineEdit">
           <property name="inputMask">
            <string>HH</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="byte4LineEdit">
           <property name="inputMask">
            <string>HH</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="byte5LineEdit">
           <property name="inputMask">
            <string>HH</string>
           </property>
           <property name="maxLength">
            <number>2</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="sendCommandButton">
         <property name="text">
          <string>Send Command</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
